.azure/build-and-deploy.yaml

```diff
@@ -31,7 +31,7 @@ variables:
   branchName: master
   group: tests-group
   customHelmOverride: |
-    config.CUSTOM=example
+    config.APP_QUEUE_DNS=issue-2635-rabbitmq
```

Comment:
is this a temporary config for testing or we want to keep it as this value?

Reply:
this file is under some special setting where any changes are ignored when we merge i believe. @keifgwinn ?

Reply:
https://github.com/3drepo/3drepo.io/blob/1d8de93c45a426bcd8c84008cef4240bb5f4abe7/.gitattributes#L8 is supposed to ignore changes to this file when doing the merge, however I noticed it didn't work last time.

Reply:
sounds like this is only triggered if there's a conflict https://stackoverflow.com/a/27234774

so it wouldn't work if the staging version was not touched as well.

Reply:
regarding the original merge, this file can be abandoned it's only for moving it to a dedicated queue.

Reply:
@Charence reverted